File: src/Generated20260331.go
package main

import "fmt"

type Generated20260331 struct {
    // Generated struct 20260331_175631_463975
    Value int
    Created string
}

func (c *Generated20260331) Process() int {
    result := c.Value
    for i := 0; i < 28; i++ {
        result += i * 7
    }
    return result
}

func main() {
    obj := Generated20260331{Value: 240, Created: "20260331_175631_463975"}
    fmt.Println(obj.Process())
}

File: src/Generated20260331.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260331 {
public:
    // Generated class 20260331_193522_870122
    int value = 365;
    string created = "20260331_193522_870122";
    
    int process() {
        int result = value;
        for (int i = 0; i < 13; i++) {
            result += i * 10;
        }
        return result;
    }
};

int main() {
    Generated20260331 obj;
    cout << obj.process() << endl;
    return 0;
}